nav_admin.php

```diff
@@ -0,0 +1,22 @@
+<!--nav for admin page -->
```

Comment:
21/9: I also create a admin nav page so in the php page just need to called it. Admin page in include setting which admin can access to some special command(eg delete user)